README.md

```diff
@@ -53,6 +53,15 @@ Calculates revenue for a list of users on a specific protocol between startTimes
 yarn ts-node ./scripts.calculateRevenue.ts --protocolId Beefy --inputAddresses addresses.csv --outputFile output.csv --startTimestamp 1739311922 --endTimestamp 1741731122
 ```
 
+### Calculate Network Revenue
+
+Calculates the revenue for a network on a specific protocol
+
+```bash
+# startBlock & endBlock are optional
+yarn ts-node ./scripts/calculateNetworkRevenue.ts --protocolId Beefy --networkId arbitrum-one
+```
+
```

Comment:
Why is this a separate script? Shouldn't this be a part of calculateRevenue where 'optimism' or 'celo' are protocol options?

Reply:
> AC:

> Integrate code with https://github.com/divvi-xyz/divvi-protocol-v0/blob/main/scripts/calculateRevenue.ts

Reply:
Based on what the AC states below it sounds like we want to provide a network, protocol ID, and optional block range. 

> Given a blockchain network (e.g., Celo, Polygon, Optimism, etc.) a frontend ID/referrer ID, and block range we're able to calculate the sum of network revenue attributable to the frontend in that block range